cdap-ui/app/cdap/components/Experiments/store/CreateExperimentActionCreator.js

```diff
@@ -574,9 +585,28 @@ function fetchAlgorithmsList() {
           algorithmsList
         }
       });
+    }, (err) => {
+      setExperimentCreateError(`Failed to fetch algorithms: ${err.response || err}`);
     });
 }
 
+function setExperimentCreateError(error) {
```

Comment:
We seem to be using this in places where we want to reset error I believe (`setExperimentCreateError.bind(null, null)`) . If we are using `.bind` operator to set null to the argument being passed can we set it as default here so that we don't have to do a bind to `null` and send the first parameter as `null`.?